front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/AdminPubsCarrousel/AdminPubCard/index.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import { Box, Text, Icon, Title } from '@citizenlab/cl2-component-library';
+
+import { IAdminPublicationData } from 'api/admin_publications/types';
+import useProjectFolderImage from 'api/project_folder_images/useProjectFolderImage';
+import useProjectFolderById from 'api/project_folders/useProjectFolderById';
+import useProjectImage from 'api/project_images/useProjectImage';
+import useProjectById from 'api/projects/useProjectById';
+
+import useLocalize from 'hooks/useLocalize';
+
+import AvatarBubbles from 'components/AvatarBubbles';
+
+import { useIntl } from 'utils/cl-intl';
+import Link from 'utils/cl-router/Link';
+import { truncate } from 'utils/textUtils';
+
+import { CardContainer, CardImage } from '../../BaseCard';
+import { CARD_WIDTH } from '../constants';
+
+import messages from './messages';
+import { getPublicationURL } from './utils';
+
+interface Props {
+  adminPublication: IAdminPublicationData;
+  ml?: string;
+  mr?: string;
+  onKeyDown?: React.KeyboardEventHandler<HTMLAnchorElement> &
+    React.KeyboardEventHandler<HTMLDivElement>;
+}
+
+interface InnerProps extends Props {
+  imageUrl?: string;
+  avatarIds: string[];
+  userCount?: number;
+}
+
+export const AdminPubCard = ({
+  adminPublication,
+  imageUrl,
+  avatarIds,
+  userCount,
+  ml,
+  mr,
+  onKeyDown,
+}: InnerProps) => {
+  const { formatMessage } = useIntl();
+  const localize = useLocalize();
+
+  const {
+    visible_children_count,
+    publication_title_multiloc,
+    publication_description_preview_multiloc,
+  } = adminPublication.attributes;
+
+  const { type } = adminPublication.relationships.publication.data;
+
+  return (
+    <CardContainer
+      as={Link}
+      tabIndex={0}
+      w={`${CARD_WIDTH}px`}
+      ml={ml}
+      mr={mr}
+      to={getPublicationURL(adminPublication)}
+      display="block"
+      onKeyDown={onKeyDown}
+    >
+      <CardImage imageUrl={imageUrl} />
```

Comment:
For accessibility, I think we need an alt label on this image right? Is that something admins can set already for project/folder images? I remember it being in development at least.